osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
[Coordinate System Wiki page](https://github.com/OpenSimulationInterface/open-simulation-interface/wiki/Coordinate-systems-and-reference-points)